code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -4,7 +4,9 @@
 
 	if(def_zone)
 		if(isbodypart(def_zone))
-			return checkarmor(def_zone, type)
+			var/obj/item/bodypart/bp = def_zone
+			if(bp.body_part)
+				return checkarmor(def_zone, type)
 		var/obj/item/bodypart/affecting = get_bodypart(ran_zone(def_zone))
 		return checkarmor(affecting, type)

```

Comment:
does this actually fix the issue? pretty sure the part that was runtiming was caused by this not checking if affecting exists

Reply:
well, i lopped off a few limbs from a human and threw a bunch of stuff at him and it didnt runtime

Reply:
ill add it to both checks just in case